src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -44,9 +46,12 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
+                $this->locale = $options['locale'];
```

Comment:
We should probably deprecate the `ChoiceLoaderInterface` implementation in those types as done in https://github.com/symfony/symfony/pull/23648.